package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "middy-koa-wrapper",
-  "version": "1.1.5",
+  "version": "1.1.6",
```

Comment:
can you remove this and let semantic release do its thing? this is a new feature not a bugfix so it should be a minor not a patch. Other than that, looks fine.

Reply:
yes, done.